tools/travis_deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+if [ -z "$TRAVIS_PULL_REQUEST" ]; then
+    echo "'TRAVIS_PULL_REQUEST' is not set."
+    exit 1
+fi
+
+
+if [ -z "$SNAP_TOKEN" ]; then
+    echo '"SNAP_TOKEN" is not set.'
+    exit 1
+fi
+
+# Login
+echo "$SNAP_TOKEN" | /snap/bin/snapcraft login --with -
```

Comment:
This will only work for PRs from this repo, right? Not forks?

Reply:
right